### ISSUE TITLE
Will pd-runner be available again?

### ISSUE THREAD

**Issue by Austrapede** [lihaoyun6/DimQQ#3]
Will pd-runner be available again?
Where can i download the latest version?

I know this is not the right place to ask, but it's the only way to contact the developer.

**Comment by lihaoyun6** [comment]
@Austrapede 
Yes, but right now it's only available on gitee (a GitHub-like code hosting site).
Since this is a site in China, I can avoid the DMCA here (until Parallels warned me again with Chinese laws).
Fortunately, since I use both GitHub and Gitee as update servers, users with v0.3.5 or later can still get updates that may exist in the future.

**Comment by lihaoyun6** [comment]
@Austrapede 
[Here](https://gitee.com/lihaoyun/PD-Runner) is the PD Runner repository on Gitee.
And [here](https://gitee.com/lihaoyun/PD-Runner/blob/main/release/release.md) is the changelog page.

**Comment by Austrapede** [comment]
@lihaoyun6
Thanks for your work! Do not give up on your endeavor.

**Comment by kainjinez** [comment]
Gitee become 404 now. Do you have any plan on other site?
@lihaoyun6

**Comment by backermanbd** [comment]
@lihaoyun6 Hi bro, pls share a copy of pd-runner.
NB: u can easily host in mega.nz or mirrored.to as well.
or mail 1 copy at megaxbang@gmail.com , i'll be more than happy!
And what is the latest version? will the releases after v3.5 have auto update features?
Regards.

**Comment by backermanbd** [comment]
> @lihaoyun6 Thanks for your work! Do not give up on your endeavor.

were u able to download a copy of pd-runner?
if, then pls don't hesitate to share with me, its a request.
i'm available at megaxbang@gmail.com

**Comment by Sakamoto-afk** [comment]
您好，请问您现在还在做这个项目吗？我之前在macwk上下载了同名的软件，不知道是不是您的。

**Comment by backermanbd** [comment]
@lihaoyun6 @Austrapede @kainjinez where can i get an original copy of pd-runner?

**Comment by backermanbd** [comment]
@Austrapede hello?

**Comment by Asura88** [comment]
https://gitee.com/lihaoyun/PD-Runner 访问受限403啊

**Comment by stzengpx** [comment]
![image](https://user-images.githubusercontent.com/5326923/168264845-5dc1b685-83ae-4067-8398-9cccd207f690.png)

**Comment by fanxiaofan577** [comment]
哥pd-runner被搞了啊

**Comment by shangyuan0818** [comment]
Gitee也没了，现在最新的macOS好像还有兼容性问题，希望能尽快修复啊

**Comment by slipttees** [comment]
Alternative [Here](https://drive.google.com/file/d/1Jh-8EGqX3lwUVev60zlrwMAgX5tbZ0zi/view?usp=sharing)

**Comment by kkksa528** [comment]
Do not need to spend time to update every month, only need to see if there is a big update on PARALLELS DESKTOP every few months, you do not need to update all the time, always use the old version on it, so that a year at most only need 3 times to update, the normal situation only need 2 times to update

**Comment by Jolkant** [comment]
**Which** version of PD runner should we use with **which** version of Parallels? I believe this all started after we updated our macs.
Also, @lihaoyun6, please contact me at your earliest convenience. parapd234@gmail.com.

**Comment by jbarlov** [comment]
Bumping / adding to the list. Any news on gitee alts?
